AuthenticationExample/AuthenticationExample/AuthenticationApp.swift

```diff
@@ -26,9 +26,9 @@ struct AuthenticationApp: App {
             // If you want to use OAuth, uncomment this code:
             //oAuthUserConfigurations: [.arcgisDotCom]
         )
-        // Set the ArcGIS and Network challenge handlers to be the authenticator we just created.
-        ArcGISEnvironment.authenticationManager.arcGISAuthenticationChallengeHandler = authenticator
-        ArcGISEnvironment.authenticationManager.networkAuthenticationChallengeHandler = authenticator
+        // Sets authenticator as ArcGIS and Network challenge handlers to handle authentication
+        // challenges.
+        ArcGISEnvironment.authenticationManager.handleAuthenticationChallenges(using: authenticator)
```

Comment:
does this compile?

Reply:
changed locally but missed to push the changes up